data/serializer/src/main/java/tech/pegasys/teku/api/schema/KZGProof.java

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
I don't know but why do we need these schema classes given we use the `jsonTypeDefinition` from the `SchemaDefinitions`?

Reply:
Maybe required for swagger-ui?

Reply:
I probably could make it Bytes

Reply:
I'd better make `Bytes48` and at least we will get one class instead of two

Reply:
My question actually is what are these schema objects under `api/schema` used for?

Reply:
Nice point, was able to remove it